src/osd/OSDMap.cc

```diff
@@ -5007,15 +5051,16 @@ class OSDUtilizationPlainDumper : public OSDUtilizationDumper<TextTable> {
 
     dump_stray(tbl);
 
+    auto dumped = get_dumped_osds();
     *tbl << ""
 	 << ""
 	 << "" << "TOTAL"
-	 << byte_u_t(pgmap.get_osd_sum().statfs.total)
-	 << byte_u_t(pgmap.get_osd_sum().statfs.get_used_raw())
-	 << byte_u_t(pgmap.get_osd_sum().statfs.allocated)
-	 << byte_u_t(pgmap.get_osd_sum().statfs.omap_allocated)
-	 << byte_u_t(pgmap.get_osd_sum().statfs.internal_metadata)
-	 << byte_u_t(pgmap.get_osd_sum().statfs.available)
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.total)
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.get_used_raw())
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.allocated)
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.omap_allocated)
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.internal_metadata)
+	 << byte_u_t(pgmap.get_osd_sum(dumped).statfs.available)
```

Comment:
instead of calling get_osd_sum() 6 times, let's call it once and put it in temp variable?

Reply:
@liewegas Updated!